Tests/UIKitTests/UIRefreshControlExntesionsTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIRefreshControlExntesionsTests.swift
+//  SwifterSwift
+//
+//  Created by ratul sharker on 7/24/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if os(iOS)
+
+import XCTest
+@testable import SwifterSwift
+
+final class UIRefreshControlExtensionTests: XCTestCase {
+
+    func testBeginRefreshProgramatically() {
+        let tableFrame = CGRect.init(x: 0, y: 0, width: 400, height: 400)
+        let tableView = UITableView.init(frame: tableFrame)
+        XCTAssertEqual(tableView.contentOffset, CGPoint.zero)
```

Comment:
Another nice cleanup you can do here is just write `XCTAssertEqual(tableView.contentOffset, .zero)`. Swift _should_ be able to understand that in this context `.zero` is a reference to `CGPoint.zero`, since you're trying to call a generic function with a `CGPoint` for its first parameter.